roles/redhat_subscription/tasks/main.yml

```diff
@@ -21,14 +21,22 @@
 # Below we use string formatting to use a variable as the name of the file in
 # the lookup()
 # via http://stackoverflow.com/a/34239020
-#
-  - name: Get origin file
-    command: ostree admin --print-current-dir
-    register: origin_file
 
-  - name: Get refspec from origin file
-    command: grep refspec {{ origin_file.stdout }}.origin
-    register: refspec
+  - name: status of /usr/bin/ostree is known
+    shell: type -Pf /usr/bin/ostree | /bin/true
```

Comment:
Okay, blocks are no problem either.  What about the idea of re-using ``atomic_host_check`` (for ``is_atomic``)?

I don't needing ``meta/main.yml``, maybe it can be worked the other way around though...

Stick the atomic-detection task under ``common`` then include that here and in ``atomic_host_check``?  (and I can throw away #160)